### PR TITLE
Added last used window position to saved settings

### PR DESCRIPTION
**Pull request by zeebok**
This pertains to #328 and adds a state to Settings for the window position. I used the default position setting from Code as I have not read about a convention for that but am open to suggestions of a better one.

### CODE REVIEW

File: schemas/io.elementary.appcenter.gschema.xml [hunk 1]
```diff
@@ -15,6 +15,16 @@
       <summary>Window height</summary>
       <description>Window height</description>
     </key>
+    <key type="i" name="window-x">
+      <default>250</default>
```
Comment (mdh34) [inline schemas/io.elementary.appcenter.gschema.xml]: I would set this to -1 as default and do some checking in the code

File: schemas/io.elementary.appcenter.gschema.xml [hunk 2]
```diff
@@ -15,6 +15,16 @@
...
+      <summary>Window X position</summary>
+      <description>Window screen position on the X-axis</description>
+    </key>
+    <key type="i" name="window-y">
+      <default>250</default>
```
Comment (mdh34) [inline schemas/io.elementary.appcenter.gschema.xml]: same here

File: src/MainWindow.vala [hunk 3]
```diff
@@ -66,6 +66,7 @@ public class AppCenter.MainWindow : Gtk.ApplicationWindow {
                 this.maximize ();
                 break;
             default:
+                move (saved_state.window_x, saved_state.window_y);
```
Comment (mdh34) [inline src/MainWindow.vala]: Add an if statement to check for defaults here so we don't move the window to 250x250 on the first launch - let the window manager handle that

**Comment by artemanufrij** [comment]
could you test your branch... after I installed it appcenter starts always on start up (elementary)

**Comment by mdh34** [comment]
@artemanufrij do you mean it sits in the dock with an update badge?

**Comment by zeebok** [comment]
I made the changes but upon testing I am also seeing the issue of the window appearing at start-up. Quick glance doesn't really give me an idea why as silent mode is suppose to skip the creation of the window. I'll take a look into this a little more tomorrow.

**Pull request merged**